samples/web/content/apprtc/room.py

```diff
@@ -65,6 +65,27 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def remove_room(host, room_id, client_id):
+  memcache_client = memcache.Client()
```

Comment:
There is actually an interesting race condition either way.

Client A calls add_client_to_room
Before A calls .cas, B calls remove_room and deletes the value (either through delete or cas set to None)
Client A fails .cas, retrieves the key again, room is None, so it recreates the room

Maybe we should mark the room as deleted on the room object, and either clean them out periodically or just let memcache evict them.